### PR TITLE
Improves quality of flavoured cigarettes

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
tweak: Robust Gold cigarettes now taste faintly of expensive metal.
tweak: Uplift Smooth's menthol effect now actually prevents you from coughing. Uplift Smooth now taste of "smoke and mint", rather than "mint and a hint of smoke".
/:cl:

So I noticed that this "anti-coughing" check in the emote code directly checked for the presence of specific reagents, which I thought was dumb. I originally had the reagent add the trait directly, but it turns out that smoking menthol cigarettes gives you very small doses of the reagent, and it doesn't stay directly in your system for long. So instead, it triggers/refreshes a 60 second status effect that gives you the TRAIT_SOOTHED_THROAT, which suppresses the *cough emote, as well as the negative effects of the Nervous Cough mental trauma.

Also, I increased the amount of gold in the Robust Gold cigs, so you can taste it now.

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
don't press blame please

**Comment by coiax** [comment]
@ExcessiveUseOfCobblestone Anything stopping this PR from being merged?

**Comment by Jordie0608** [comment]
Yes conflicts are.

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -270,14 +270,14 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 
 /obj/item/clothing/mask/cigarette/uplift
 	desc = "An Uplift Smooth brand cigarette."
-	list_reagents = list("nicotine" = 7.5, "menthol" = 7.5)
+	list_reagents = list("nicotine" = 13, "menthol" = 5)
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/cigs_lighters.dm]: undocumented?

Comment (coiax) [reply code/game/objects/items/cigs_lighters.dm]: Ah yeah sorry, I forgot to update the CL. It's in my commit titles though:
Menthol cigs taste equally of smoke and mint while still buffing

**Pull request merged**